### PR TITLE
chore(deps): update dependencies version of ngrx/store

### LINKED ISSUE #37: Required Ngrx/store version 6.0.0-beta.1
Is there a reason why ngrx-store-freeze is still requiring `@ngrx/store` beta version?

https://github.com/brandonroberts/ngrx-store-freeze/blob/79cac4ee7d75a89fe3d4a28ddb5225d315e2f9f7/package.json#L53

### PR DESCRIPTION
**Pull request by MattiJarvinen**
Closes #37

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,14 +43,14 @@
   },
   "homepage": "https://github.com/brandonroberts/ngrx-store-freeze#readme",
   "peerDependencies": {
-    "@ngrx/store": ">=4.0.0 || ^6.0.0-beta.1"
+    "@ngrx/store": ">=4.0.0 || ^6.0.0"
```
Comment (brandonroberts) [inline package.json]: Just having `>=4.0.0` is sufficient to cover 6.x

**Review by brandonroberts — CHANGES_REQUESTED** [review]
Delete the package-lock.json file also

File: package.json (hunk 1 above)
Comment (MattiJarvinen-BA) [reply package.json]: Ah... what was I thinking needs more ☕️

**Comment by brandonroberts** [comment]
No, don't delete either one. I thought there was only a `yarn.lock` file and not both

**Comment by MattiJarvinen-BA** [comment]
Package-lock.json isn't old though but ok.

**Review by brandonroberts — APPROVED** [review]

**Comment by brandonroberts** [comment]
Published as `0.2.4`

**Pull request merged**